weave-js/src/components/PagePanelComponents/Home/Browse2/SmallRef.tsx

```diff
@@ -76,11 +78,92 @@ export const objectRefDisplayName = (
   throw new Error('Unknown ref type');
 };
 
-export const SmallRef: FC<{
-  objRef: ObjectRef;
-  wfTable?: WFDBTableType;
+export const SmallRefBox: FC<{
+  iconName: IconName;
+  text: string;
+}> = ({iconName, text}) => (
+  <Box display="flex" alignItems="center">
+    <Box
+      mr="4px"
+      bgcolor={hexToRGB(MOON_300, 0.48)}
+      sx={{
+        height: '22px',
+        width: '22px',
+        borderRadius: '16px',
+        display: 'flex',
+        flex: '0 0 22px',
+        justifyContent: 'center',
+        alignItems: 'center',
+      }}>
+      <Icon name={iconName} width={14} height={14} />
+    </Box>
+    <Box
+      sx={{
+        height: '22px',
+        flex: 1,
+        minWidth: 0,
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+      }}>
+      {text}
+    </Box>
+  </Box>
+);
+
+export const SmallArtifactRef: FC<{
+  objRef: WandbArtifactRef;
   iconOnly?: boolean;
-}> = ({objRef, wfTable, iconOnly = false}) => {
+}> = ({objRef}) => {
+  // TODO lookup the artifact by name (where name is name:version and version is "v0", or "v1", etc)
+  // so you can verify it's a valid artifact AND check that 'model' is the right type
+  /*
+  const ARTIFACT_BY_NAME_QUERY = gql`
+    query ArtifactByName(
+      $entityName: String!,
+      $projectName: String!,
+      $name: String!
+    ) {
+      project(name: $projectName, entityName: $entityName) {
+        artifact(name: $name) {
+          id
+          name
+          description
+          createdAt
+          updatedAt
+          size
+          state
+        }
+      }
+    }
+  `;
+  */
+
+  return (
+    <Link
+      $variant="secondary"
+      style={{
+        width: '100%',
+        minHeight: '38px',
+        display: 'flex',
+        alignItems: 'center',
+      }}
+      as="a"
+      href={`${window.location.origin}/${objRef.entityName}/${objRef.projectName}/artifacts/model/${objRef.artifactName}/${objRef.artifactVersion}`}
```

Comment:
the hardcoded `artifacts/model` takes you to the artifacts registry in the models folder, at the screenshot below for example

if you ever wanted to reference another type of artifact (ie, `prompt` from below), you'd either have to 
- make a call to gorilla in this component to lookup the artifact by name, and get more metadata to construct the link

_OR_

- pass the artifact type as part of the artifact URI when creating the weave object (in your python script that's calling the weave SDK code)

 
<img width="1000" alt="image" src="https://github.com/user-attachments/assets/816c37da-8204-4c41-a018-cfd49e33b1eb">